dependabot_api.py

```diff
@@ -30,7 +28,7 @@ def get_repos_by_status(org):
         if account.attributes["github-login"] == org:
             for state in states:
```

Comment:
Do we need to loop through the states if we are only using active_repos now or are we leaving this here in case we can go back to the original endpoint?

Reply:
You found the bug! That's why we had to rewrite the other function. I've removed the different `states` for dependabot and cleaned up the references to that state.

Reply:
![](https://gph.is/297cNL0)

Reply:
![](https://gph.is/297cNL0)